MXClient-php/docs/Api/AccountsApi.md

```diff
@@ -0,0 +1,218 @@
+# atrium\AccountsApi
+
+Method | HTTP request | Description
+------------- | ------------- | -------------
+[**listAccountTransactions**](AccountsApi.md#listAccountTransactions) | **GET** /users/{user_guid}/accounts/{account_guid}/transactions | List account transactions
+[**listUserAccounts**](AccountsApi.md#listUserAccounts) | **GET** /users/{user_guid}/accounts | List accounts for a user
+[**readAccount**](AccountsApi.md#readAccount) | **GET** /users/{user_guid}/accounts/{account_guid} | Read an account
+[**readAccountByMemberGUID**](AccountsApi.md#readAccountByMemberGUID) | **GET** /users/{user_guid}/members/{member_guid}/accounts/{account_guid} | Read an account
+
+
+# **listAccountTransactions**
+> \atrium\model\TransactionsResponseBody listAccountTransactions($account_guid, $user_guid, $from_date, $to_date, $page, $records_per_page)
+
+List account transactions
+
+This endpoint allows you to see every transaction that belongs to a specific account. The default from_date is 90 days prior to the request, and the default to_date is 5 days from the time of the request.<br> The from_date and to_date parameters can optionally be appended to the request.
+
+### Example
+```php
+<?php
+require_once(__DIR__ . '/vendor/autoload.php');
+
+// Configure API Key authorization
+$config = atrium\Configuration::getDefaultConfiguration()->setHeader('MX-API-Key', 'YOUR_API_KEY');
```

Comment:
Can we configure multiple headers at once? That would be nice to have.

Reply:
I believe that method returns `$this` so we can do $x->setHeader('MX-API-Key', 'YOUR_API_KEY')->setHeader('something_else', 'blah');`

Reply:
Good idea. We did something similar in the Go library. I've just updated it so we pass the api key and client id in when creating an AtriumClient.

```
$client = new atrium\Api\AtriumClient(
    "YOUR_API_KEY",
    "YOUR_CLIENT_ID",
    new GuzzleHttp\Client()
);
```